packages/cli/templates/default-backend-plugin/package.json.hbs

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "@backstage/cli": "{{versionQuery '@backstage/cli'}}",
     "@types/supertest": "{{versionQuery '@types/supertest' '2.0.12'}}",
+    "eslint-plugin-jest": "{{versionQuery 'eslint-plugin-jest' '27.1.6'}}",
```

Comment:
I can take a look, but my nodejs tooling knowledge is not as good as it is for other languages

Reply:
it would seem like adding this would also work - but it's dependent on filesystem path

```
❯ git --no-pager diff       
diff --git plugins/plugin-backen/.eslintrc.js plugins/plugin-backend/.eslintrc.js
index e2a53a6..a9a1fbe 100644
--- plugins/plugin-backend/.eslintrc.js
+++ plugins/plugin-backend/.eslintrc.js
@@ -1 +1,3 @@
-module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+  extends: ['../../.eslintrc.js'],
+});
```

Reply:
I had another look at this and it turns out ESLint doesn't support my suggested solution of resolving absolute paths, or more like actively refuses it. Right now the proposed solution is to use peer dependencies for plugins and then tools to automatically install peer dependencies, I really don't want to do that because that'll mess with other deps 😅. A possible solution could be to separate out the ESLint config into its own package as that might flatten the dependency graph in some useful ways, but not sure.

Either way, it turns out that the people over at ESLint have been working on a new an improved config format that as far as I can tell will solve this issue and a lot of the other ones that we're seeing when it comes to configuring ESLint for a monorepo. The RFC is over at https://github.com/backstage/backstage/pull/15050, and implementation is underway in https://github.com/eslint/eslint/issues/13481. Any changes we do to the way our ESLint config is packages and set up should be taking this new config format into account.